spec/requests/swagger_docs/v5/full_assessment_spec.rb

```diff
@@ -433,7 +433,7 @@
                                   description: "Date payment made",
                                   example: "1992-07-22",
                                 },
-                                housing_costs_type: {
+                                housing_cost_type: {
                                   type: :string,
                                   enum: CFEConstants::VALID_OUTGOING_HOUSING_COST_TYPES,
                                   description: "Housing cost type (omit for non-housing cost outgoings)",
```

Comment:
I thought you were going to add a 'oneOf' so that this bit of the API is documented in a better fashion?

Reply:
But the issue was a typo in the field name preventing it from being saved?